docs/guides/Redux-Integration.md

```diff
@@ -2,16 +2,16 @@
 
 To handle your app's navigation state in redux, you can pass your own `navigation` prop to a navigator. Your navigation prop must provide the current state, as well as access to a dispatcher to handle navigation options.
 
-With redux, your app's state is defined by a reducer. Each navigation router effectively has a reducer, called `getStateForAction`. The following is a minimal example of how you might use navigators within a redux application:
+With redux, your app's state is defined by a reducer. Each navigation router effectively has a reducer, called `getStateForAction`. You can dispatch `NavigationActions.init` to initialize your router state.
+
+The following is a minimal example of how you might use navigators within a redux application:
 
 ```
-import { addNavigationHelpers } from 'react-navigation';
+import { addNavigationHelpers, NavigationActions } from 'react-navigation';
 
 const AppNavigator = StackNavigator(AppRouteConfigs);
 
-const initialState = AppNavigator.router.getStateForAction(AppNavigator.router.getActionForPathAndParams('Login'));
```

Comment:
To be clear, the failure with navigate was somewhere in RNav, trying to access the previous route but failing to access the nonexistent index.

I also tried to set an initial route of `{index: -1, routes: []}`, but that didn't work either.

It would be nice if navigate handled this initial null/empty case better, but it's not really intended for that. Init explicitly is intended for that.